src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+    operate(callAction, num) {
+      callAction(num);
+
+      return calc;
+    },
+
+    add(prop) {
+      calc.result += prop;
```

Comment:
better to use `this` here instead of the calc, it will improve the reusability

Reply:
try to use 'this' here instead calc and after we will discuss why is no working properly